angular/build/config/vendor.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Open to moving the file, but we need it because we need a place to specify what files are included in vendor.js

There's also a .gitignored file in there called `plugins.yml` which is autogenerated and does the same thing for plugins (ie, include components from the plugins into app.js).

Once gulp compile is run, everything in /build could be rm -rfed and everything would work fine